helpers/async.ts

```diff
@@ -0,0 +1,15 @@
+import * as glob from 'glob';
```

Comment:
nit: So does this repo not use the copyright header consistently?

Reply:
Oh, it's pretty consistent about *not* using it. :-)

Reply:
On my list.